__tests__/Record.ts

```diff
@@ -88,6 +88,12 @@ describe('Record', () => {
     expect(t1.equals(null)).toBeFalsy();
   });
 
+  it('if compared against Map should return false', () => {
+    const MyType = Record({ a: 1, b: 2 });
+    const t1 = MyType();
+    expect(t1.equals(Map({ a: 1, b: 2 }))).toBeFalsy();
```

Comment:
For completeness can you add a test which flips the operands to ensure the symmetric property?

Reply:
Sure